projects/cn0506/zc706/system_project.tcl

```diff
@@ -34,7 +33,7 @@ adi_project cn0506_zc706 0 [list \
   INTF_CFG  $intf \
 ]
 
-adi_project_files  cn0506_zc706 [list \
```

Comment:
On all cn0506 projects the folders are not created properly. If I run:
`
make INTF_CFG=RMII
`
created folder is just
`INTF`
but I'm expecting 
`INTFCFGRMII`

I'm putting the comment here since that's reproducible only on cn0506 projects (even if I'm sure the root cause is not here).

Reply:
Should be fixed now